scripts/ngram-count-patterns-maxent2.py

```diff
@@ -178,10 +178,11 @@ def wrap (x):
 						meoutcomes[slword][event_counter] = '';
 					#}
 					for ni in ngrams[slword]: #{
-						if ni not in features: #{
+						if ni not in features: #and ni != "": {
```

Comment:
what happens when ni=="" here?

Reply:
oops!! I have to remove that. It was sort of workaround for some error which I am no longer getting :grin: :grin: